source/documentation/deploying_services/elasticsearch.md

```diff
@@ -140,7 +140,7 @@ where `SERVICE_NAME` is a unique descriptive name for this service instance, and
 cf update-service my-elasticsearch-service -p small-ha-7.x
 ```
 
-The plan upgrade will start immediately and finish within an hour. You can check the status of the upgrade by running `cf services`.
+The plan upgrade will start immediately and finish within an hour, except for large plans which may take over an hour. You can check the status of the upgrade by running `cf services`.
```

Comment:
It might be worth giving an example of what we mean by 'large' - what kind of size, exactly?

Reply:
I think this refers to `large` and `xlarge` plans.